sandbox/registration-processor-dmz.properties

```diff
@@ -163,6 +166,13 @@ mosip.registration.processor.registration.status.version=1.0
 mosip.registration.processor.lostrid.version=1.0
 # Registration External Status Version
 mosip.registration.processor.registration.external.status.version=1.0
+#Packet Status Version
+mosip.registration.processor.packet.external.status.version=1.0
+#Packet Status Id
+mosip.registration.processor.packet.external.status.id=mosip.registration.packet.external.status
+mosip.registration.processor.packet.status.transactiontypecodes-before-uploading-to-objectstore=PACKET_RECEIVER,SECUREZONE_NOTIFICATION
+mosip.registration.processor.packet.status.transactiontypecodes-uploading-to-objectstore=UPLOAD_PACKET
+mosip.registration.processor.packet.status.transactiontypecodes-time-based-resend-required=PACKET_RECEIVER
```

Comment:
1. Move "mosip.registration.processor.packet.external.status.id" after the mosip.registration.processor.registration.external.status.id property
2. In comments always give space between # and first letter
3. Have one empty line between version properties and above 3
4. Please add comments for each of the 3 properties

Reply:
updated

Reply:
resolved